src/app/frontend/_variables.scss

```diff
@@ -31,6 +31,7 @@ $primary: #326de6;
 $secondary: #f51200;
 $warning: #ffa500;
 $delicate: #aaa;
+$border: #ddd;
```

Comment:
All of `foreground-X` have alpha factor, which can cause overlays in few scenarios. To make borders color always the same I've added new variable.